src/main/java/seedu/address/model/person/Person.java

```diff
@@ -147,4 +147,35 @@ public String toString() {
         return builder.toString();
     }
 
+    /**
+     * Returns the formatted export string for each Person.
+     */
+    public String toExportFormat() {
+        final StringBuilder builder = new StringBuilder();
+        builder.append("Name: ")
+            .append(getName())
+            .append("\nCategory: ")
+            .append(getCategoryCode())
+            .append("\nPhone: ")
+            .append(getPhone())
+            .append("\nEmail: ")
+            .append(getEmail())
+            .append("\nAddress: ")
+            .append(getAddress())
+            .append("\nReview: ")
+            .append(getReview());
+
```

Comment:
I think the new export format looks great!

Reply:
thanks!